Assets/Scripts/PyraController.cs

```diff
@@ -1,13 +1,17 @@
+using Constants;
 using Projectiles;
 using UnityEngine;
+using UnityEngine.SceneManagement;
 
-public class Player : MonoBehaviour
+public class PyraController : MonoBehaviour
 {
     [SerializeField] private float speed = 5.0f;
+    [SerializeField] public float _phealth = 100.0f;
```

Comment:
Nici câmpurile public, nici alea serializate nu se prefixează cu _, deci oricum o dai _ n-are ce căuta aici.

`phealth` reprezintă 2 "cuvinte": p și health, deci ar fi trebuit să fie lowerCamelCase: pHealth. p e redundant, pentru că e clar că e viața playerului ținând cont că ăsta e scriptul de player, deci `health` ar fi ajuns.